docs/src/tutorials/standalone/Usage/LSM_single_column_tutorial.jl

```diff
@@ -1,5 +1,5 @@
 # The `AbstractModel`
-# [tutorial](https://clima.github.io/ClimaLand.jl/stable/generated/model_tutorial)
+# [tutorial](@ref "General setup")
 # describes how a user can run
 # simulations of a physical system governed by differential equations.
 # In this framework, the user must define a model type for their problem,
```

Comment:
Minor, but for the soil model code link, could we link to the richards equation file instead?
rre.jl

Reply:
Will do!